### DISCUSSION TITLE
(Not sure where to put this) Downloads on website fail due to invalid cert

### DISCUSSION THREAD

**Discussion by brisingraerowing** [ImageMagick/ImageMagick#3873] [General]
I went to download the Win64 portable version from the website, and got a certificate error stating that the certificate is only valid for `imagemagick.net`.

**Comment by dlemstra** [comment]
We are in the middle of maintenance of our servers. When this is done you will be able to download those files again.

**Reply by brisingraerowing** [reply]
Ah. Good to know.
